nextcord/audit_logs.py

```diff
@@ -549,6 +548,9 @@ def created_at(self) -> datetime.datetime:
 
     @utils.cached_property
     def target(self) -> AuditTarget:
+        """:class:`Any`: The target that got changed.
```

Comment:
Can you modify this from `Any` to the actual return type please?

Reply:
Yes, but this union is quite long, I don't think it is necessary.
AuditTarget = `Union[Guild, abc.GuildChannel, Member, User, Role, Invite, Emoji, StageInstance, GuildSticker, Thread, Object, AutoModerationRule, None]`

Reply:
Can you not just :class:`AuditTarget`

Reply:
I see, hrmmmm

Reply:
Yes, so I would leave it as it is